.readthedocs.yml

```diff
@@ -4,7 +4,7 @@
 version: 2
 
 python:
-  version: 3.7
+  version: 3.10
```

Comment:
See my main comment about deleting this file altogether, but for future reference, you annoyingly have to quote "3.10" so it does not get interpreted as the float `3.1`.